tcl/dialog_iemgui_knob.tcl

```diff
@@ -0,0 +1,175 @@
+# For information on usage and redistribution, and for a DISCLAIMER OF ALL
+# WARRANTIES, see the file, "LICENSE.txt," in this distribution.
+# Copyright (c) 1997-2009 Miller Puckette.
```

Comment:
Wouldn't this be your copyright? You are the author here. If not, you could adopt a more general line such as "Copyright (c) 2020 Pure-Data team." etc.

Reply:
yep i'm going to update this.

Reply:
done